unordered_map/clear.md

```diff
@@ -0,0 +1,34 @@
+/*
+    Author : Amr Akmal
+    Date : 07/01/2020
+    Time : 19:35
+    Description : clear in an unordered_map.
+*/
+
```

Comment:
Author details are required in code files (for ex clear.cpp) only.

Reply:
Hope you clarify more what's needed to be done?

Thanks.